pynecone/pc.py

```diff
@@ -63,7 +75,17 @@ def run(
         env: The environment to run the app in.
         frontend: Whether to run the frontend.
         backend: Whether to run the backend.
+
+    Raises:
+        Exit: If the app is not initialized.
     """
+    # Check that the app is initialized.
+    if not utils.is_initialized():
+        utils.console.print(
+            f"[red]The app is not initialized. Run [bold]pc init[/bold] first."
```

Comment:
Awesome I know a couple people brought this up aswell

Reply:
Yup, we can just catch any of these small errors over time - if one person had them, I'm sure many will...